packages/turf-clean-coords/index.ts

```diff
@@ -60,7 +60,7 @@ function cleanCoords(
       var existing: Record<string, true> = {};
       getCoords(geojson).forEach(function (coord: any) {
         var key = coord.join("-");
-        if (!existing.hasOwnProperty(key)) {
+        if (!Object.prototype.hasOwnProperty.call(existing, key)) {
```

Comment:
I'm surprised it's requiring you to use hasOwnProperty in this way?

Reply:
Its a defensive change against someone modifying .hasOwnProperty on your object. See: https://eslint.org/docs/rules/no-prototype-builtins